rpm/Makefile

```diff
@@ -1,11 +1,12 @@
+ALPINE:=$(shell $(CURDIR)/../detect_alpine_image)
```

Comment:
this isn't needed by this PR (and I'll make a new one if you want) but fixes non-x86 rpm builds